tests/openqa/osautoinst/start_test.pm

```diff
@@ -19,7 +19,9 @@ sub run {
     my $arch       = get_required_var('ARCH');
     my $ttest      = 'minimalx';
     my $openqa_url = get_var('OPENQA_HOST_URL', 'https://openqa.opensuse.org');
-    my $cmd        = <<"EOF";
+    # aarch64 does not support nested virt yet, so disable kvm in this case
```

Comment:
Why disabling the HUGE_PAGES for aarch64? aren't those set somewhere else already?

Reply:
The test clone an existing job from o3 where we have KVM and HUGE_PAGES available to the tested openqa instance running in this qemu test, where we have no KVM (no nested) and no huge pages configured.

Reply:
@ggardet Ahh! cool